.deployment.json

```diff
@@ -0,0 +1,29 @@
+{
+  "env": {
+    "name": "discord-bots",
```

Comment:
this shouldn't be the env name! it will clash with our test env

Reply:
fixed